Library/Homebrew/cask/cask.rb

```diff
@@ -304,7 +297,8 @@ def artifacts_list
       artifacts.map do |artifact|
         case artifact
         when Artifact::AbstractFlightBlock
-          artifact.to_h
+          # Only indicate whether this block is used as we don't load it from the API
```

Comment:
Could do similarly for `languages`

Reply:
I think that including the languages in the JSON still is useful for users even if we don't use it.

Reply:
@apainintheneck Given no users ever requested it and it increases the size of the JSON: I disagree.